### DISCUSSION TITLE
Build -> UCI -> UCI (sunbird github release  tag not found)

### DISCUSSION THREAD

**Discussion by srikanth-6520** [Sunbird-Ed/Community#495] [Bugs] [answered]
Hello @gandham-santhosh , we have tried to build the UCI job with release tag "release-5.1.0" , that branch or tag is not available in sunbird public repository, so please  share the proper  release tag.
for you are reference , below we have shared the console output.

Started by user [Kolluri Srikanth](https://jenkins.alex-sb.dev.verlab.io/user/srikanthk)
Rebuilds build #[1](https://jenkins.alex-sb.dev.verlab.io/job/Build/job/UCI/job/UCI/1)
Checking out git https://github.com/samagra-comms/uci-apis.git into /var/lib/jenkins/workspace/Build/UCI/UCI@script/9ce893f6efb7f02da4085d25e800712a05a1f26d63a42a171519fc6d27604354 to read build/Jenkinsfile
The recommended git tool is: NONE
No credentials specified
Cloning the remote Git repository
Cloning repository https://github.com/samagra-comms/uci-apis.git
 > git init /var/lib/jenkins/workspace/Build/UCI/UCI@script/9ce893f6efb7f02da4085d25e800712a05a1f26d63a42a171519fc6d27604354 # timeout=10
Fetching upstream changes from https://github.com/samagra-comms/uci-apis.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/samagra-comms/uci-apis.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/samagra-comms/uci-apis.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > git rev-parse origin/release-5.1.0^{commit} # timeout=10
 > git rev-parse release-5.1.0^{commit} # timeout=10
ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.
ERROR: Maximum checkout retry attempts reached, aborting
Finished: FAILURE

**Comment by santhosh-tg** [comment]
@srikanth-6520  Please use this tag **release-5.0.0_RC3** . Can you share the documentation link that you are following pls ?

**Reply by Krishnaj20** [reply]
@srikanth-6520 - Are you are trying to install SunbirdED or Sunbird UCI building block? It will be helpful if you can also let us know what are you trying to enable by installing UCI so that we can provide the right guidance.

@chitranshu-keshav

**Reply by srikanth-6520** [reply]
@Krishnaj20 we are trying to install SunbirdED , and here is documentation link.
https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0/build

**Reply by Krishnaj20** [reply]
thanks @srikanth-6520 . Can you let us know what use cases you are planning to implement using SunbirdED?

**Reply by srikanth-6520** [reply]
thank you @santhosh-tg , tag "release-5.0.0_RC3", it's working  , we are able to build the job successfully.

**Reply by santhosh-tg** [reply, marked as answer]
@srikanth-6520  request you to mark this thread as answered.
